external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -53,6 +52,22 @@ def __init__(self, task_environment: TaskEnvironment):
         self._should_stop = False
         super().__init__(TASK_CONFIG, task_environment)
 
+        self._task_environment = task_environment
+        if len(task_environment.get_labels(False)) == 1:
+            self._labels = task_environment.get_labels(include_empty=True)
+        else:
+            self._labels = task_environment.get_labels(include_empty=False)
+        self._multilabel = len(task_environment.label_schema.get_groups(False)) > 1 and \
+                           len(task_environment.label_schema.get_groups(False)) == \
```

Comment:
This variable has a slightly complicated algorithm to determine whether the task is multilabel or not. So, I think the current shape of this line is reasonable to improve readability. Instead of weird shape to avoid flake8 warning, I added # noqa:E127